components/Schedule/ScheduleItemArray.jsx

```diff
@@ -0,0 +1,96 @@
+export const ScheduleItemArray = new Map 
+([
+    [
+        "Events",
+        [
+            {
+                "Check-In": 
+                {
+                    date: "Saturday, April 2", 
+                    time: "7 - 8 AM", 
+                    location: "Hybrid (In-Person @ MSE 116)"
+                },
+                "Opening Ceremony": 
+                {
+                    date:"Saturday, April 2", 
+                    time: "8 - 9 AM", 
+                    location:"Hybrid (In-Person @ MSE 116)"
+                },
+                "Red Bull Giveaway": 
+                {
+                    date:"Sunday, April 3", 
+                    time: "8 AM - 1 PM", 
+                    location:"In-Person @ WCH 205/206"
+                },
+                "Hackathon End": 
+                {
+                    date:"Sunday, April 3", 
+                    time: "9 AM", 
+                    location:""
+                },
+            },
+        ]
+    ],
+    [
+        "Workshops",
+        [
+            {
```

Comment:
I think the design of this data structure needs to change,

overall, we're using a map, which is really nice. Maps have a [key: value] pair.

our keys are: ["Events", "Workshops", "Activities"]
Each key needs to map to an array of descriptions.
Each description will have the following properties:
```
type Description = {
    eventName: string
    date: string
    time: string
    location: string
}
```
so overall we have this:
```
Map([
    ["Events": [
        {
            eventName: "EventName here"
            date: "date here",
            time: "time here",
            location: "location here",
        },
        {
            eventName: "EventName here"
            date: "date here",
            time: "time here",
            location: "location here",
        },
    ]],
    ["Workshops", [
        {
            eventName: "EventName here"
            date: "date here",
            time: "time here",
            location: "location here",
        },
        {
            eventName: "EventName here"
            date: "date here",
            time: "time here",
            location: "location here",
        },
    ]],
])
```